lib/src/layer/tile_layer/tile_image_manager.dart

```diff
@@ -117,8 +114,9 @@ class TileImageManager {
 
   void reloadImages(
     TileLayer layer,
-    TileBounds tileBounds,
-  ) {
+    TileBounds tileBounds, {
+    bool Function(TileImage tileImage)? test,
```

Comment:
Should probably rename this to be clearer what it does (I know it's internal only, but still should have a more self-explanatory name). Perhaps use [`@visibleForTesting`](https://api.flutter.dev/flutter/meta/visibleForTesting-constant.html)?

Reply:
This is not related to testing, it's an optional filter. I followed the naming convention that I've often seen in dart e.g: https://api.flutter.dev/flutter/dart-core/Iterable/where.html